cmd/flakechecker/bq_client.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"cloud.google.com/go/bigquery"
+	"context"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/api/iterator"
+	"time"
+)
+
+const projectID = "acs-san-stackroxci"
+const queryTimeout = 1 * time.Minute
+const queryStrGetFailureRatio = `
+SELECT
+    JobName,
+    Name,
+    Classname,
+    TotalAll,
+    FailRatio
+FROM
+` + "`acs-san-stackroxci.ci_metrics.stackrox_tests__recent_flaky_tests`" + `
+WHERE
+    JobName = @jobName
+    AND Name = @name
+    AND Classname = @classname
+`
+
+type biqQueryClient interface {
+	GetRatioForTest(flakeTestConfig *flakeDetectionPolicy, testName string) (int, int, error)
+}
+
+type biqQueryClientImpl struct {
```

Comment:
Just a thought: as opposed to mock client.
```suggestion
type realBigQueryClient struct {
```